dashing/distribution.yaml

```diff
@@ -184,6 +184,25 @@ repositories:
       url: https://github.com/osrf/osrf_pycommon.git
       version: master
     status: maintained
+  osrf_testing_tools_cpp:
+    doc:
+      type: git
+      url: https://github.com/osrf/osrf_testing_tools_cpp.git
+      version: master
```

Comment:
will this get a `dashing` branch in the future?

Reply:
Yeah, when the actual dashing release is out we can cut all the dashing branches. I am not volunteering to keep dashing and master branches in sync before then.